src/app/data-planes/views/DataPlaneListView.vue

```diff
@@ -311,6 +310,15 @@ const props = defineProps<{
   text-decoration: none;
 }
 
+search {
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  align-items: stretch;
+  flex-wrap: wrap;
+  gap: $kui-space-60;
+  margin-bottom: $kui-space-60;
+}
```

Comment:
diff of this and https://github.com/kumahq/kuma-gui/pull/3089/files?w=1#diff-97bc6cd40e1fca51a1e241ad46d3046f7f2b245842eccd628ed7f07c0e1e1368L153-L159

![image](https://github.com/user-attachments/assets/4fdddafe-793a-42e8-94f2-bfc8182225ca)